docs/modules/operation/pages/provisioning/hardware-inventory.adoc

```diff
@@ -0,0 +1,111 @@
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.3"  type="string" alias="ceAssetOrderablePartNumber" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.4"  type="string" alias="ceAssetHardwareRevision" replace="entPhysicalHardwareRev" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.5"  type="string" alias="ceAssetMfgAssyNumber" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.6"  type="string" alias="ceAssetMfgAssyRevision" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.7"  type="string" alias="ceAssetFirmwareID" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.8"  type="string" alias="ceAssetFirmwareRevision" replace="entPhysicalFirmwareRev" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.9"  type="string" alias="ceAssetSoftwareID" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.10" type="string" alias="ceAssetSoftwareRevision" replace="entPhysicalSoftwareRev" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.11" type="string" alias="ceAssetCLEI" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.12" type="string" alias="ceAssetAlias" />
+</hw-extension>
+----
+
+In this example, only the nodes that match the `sysOidMask .1.3.6.1.4.1.9` (Cisco Devices) are candidates to discover the `mibObj` entities defined on the extension group.
+The definition of `ceAssetSerialNumber` has been configured to replace one of the base attributes `entPhysicalSerialNum` so the adapter will know to override the generic ENTITY-MIB attribute instead of creating a new custom attribute entry.
+
+Once your `snmp-hardware-inventory-adapter-configuration.xml` file has been configured, the Provisiond service will start attempting to collect hardware inventory data when refreshing requisitions.
+After synchronizing the requisition, you should be able to see a `uei.opennms.org/internal/discovery/hardwareInventorySuccessful` event to confirm inventory was collected.
+This event will be triggered only when the provisioning adapter detects changed hardware inventory for a node.
+If for some reason, there is a problem retrieving or storing the inventory data, a `uei.opennms.org/internal/discovery/hardwareInventoryFailed` event will be sent instead.
+
+== Viewing collected hardware inventory information
+
+To view inventory information collected for a node, click the "Hardware Info" link on the node's page in the Web UI.
+If the node has hardware inventory, {page-component-title} will display all collected fields.
+
+TIP: It is also possible to retrieve and update hardware inventory information via the xref:development:rest/nodes.adoc#nodes-rest[Nodes REST endpoint].
+See the REST documentation for more information.
+
+== Using hardware information in events and notifications
+
+A set of placeholders are available in {page-component-title} to enrich the content of the events, alarms, and notifications.
+Because of the hierarchical nature of the hardware entities, you must provide two fields in order to obtain the value of the required attribute.
```

Comment:
```suggestion
Because of the hierarchical nature of the hardware entities, you must provide two fields to obtain the value of the required attribute.
```